### PR TITLE
Retail Ref Arch Update

### LINKED ISSUE #300: Documentation: Add MC4R Visio diagram in repo
### Documentation Issue

Documentation: Add MC4R Visio diagram in repo.

cc: @victorar 

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/microsoft/industry/blob/main/CODE_OF_CONDUCT.md)

### PR DESCRIPTION
**Pull request by sasever**
## Summary of the Pull Request
Graphic Design Suggestions and Changes in the MC4R scope is reflected to the design. 

## PR Checklist
* [x] Closes Issue #300
* [x] Code of Conduct signed. If not, go to [Code of Conduct](https://github.com/microsoft/industry/blob/main/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: retail/referenceImplementation/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Reference architecture for Microsoft Cloud for Retail
+# Reference architecture for Microsoft Cloud for RetailReference architecture for Microsoft Cloud for Retail
```
Comment (marvinbuss) [inline retail/referenceImplementation/README.md:1]:
```suggestion
# Reference architecture for Microsoft Cloud for Retail
```

**Review by marvinbuss — CHANGES_REQUESTED** [review]
Just one small typo. Please check below.

File: retail/referenceImplementation/README.md (hunk 1 above)
Comment (sasever) [reply retail/referenceImplementation/README.md]: oh must have pasted accidentally , good catch

**Review by marvinbuss — APPROVED** [review]
LGTM!

**Pull request merged**